CHANGELOG.md

```diff
@@ -17,9 +17,13 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.44.2] - 2024-05-30
+### Added
+- `RawRowsAPI.insert_dataframe` now has a new `dropna` setting (defaulting to True, as this would otherwise raise later).
+
 ## [7.44.1] - 2024-05-24
 ### Added
-- Missing parameter `timeout` to `client.transformations.preview`. 
+- Missing parameter `timeout` to `client.transformations.preview`.
```

Comment:
Time for an end-of-line linter?

Reply:
I think it will screw up a couple of the tests that rely on output matching. This change here is done automatically by my editor 😅